sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -2282,6 +2282,227 @@ pi_result piextKernelGetNativeHandle(pi_kernel kernel,
   return piextGetNativeHandle(kernel, nativeHandle);
 }
 
+// command-buffer extension
+pi_result piextCommandBufferCreate(pi_context context, pi_device device,
+                                   const pi_ext_command_buffer_desc *desc,
+                                   pi_ext_command_buffer *ret_command_buffer) {
+  (void)context;
+  (void)device;
+  (void)desc;
+  (void)ret_command_buffer;
+
+  // Not implemented
+  return {};
```

Comment:
Should these return some error instead? Otherwise it might look like everything is okay and the resulting bugs might be hard to trace.

Reply:
Indeed, overlooked that there is no `die()` in this adapter. Updated to returned `PI_ERROR_INVALID_OPERATION` as it's what the other unimplemented entry-points in `pi_opencl.cpp` do.